src/builder.rs

```diff
@@ -69,14 +69,40 @@ impl Builder {
             .as_mut()
             .expect("`Latency` missing")
             .min_message_latency = value;
+        // If the min latency is greater than the max latency, update
```

Comment:
I think it'd be simpler to just assert the min/max are valid on build and not bump up the max here as well.

Reply:
Yeah that sounds better, I'll cancel this change and file an updated PR!